src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,145 +1,177 @@
-      .whileTrue(new setupControls());
+    Trigger intakeMoveUp = m_operatorHID.axisGreaterThan(1, Constants.DEADZONE);
+    Trigger intakeMoveDown = m_operatorHID.axisLessThan(1, -Constants.DEADZONE);
+    intakeMoveUp.whileTrue(new InstantCommand(() -> m_intake.setRaiseIntakeSpeed(0.1), m_intake));
+    intakeMoveDown.whileTrue(new InstantCommand(() -> m_intake.setRaiseIntakeSpeed(-0.1), m_intake));
+    intakeMoveUp.or(intakeMoveDown).onFalse(new InstantCommand(() -> m_intake.setRaiseIntakeSpeed(0), m_intake));
 
-    
+
+
+    // ================DRIVER CONTROLS==========================================
+    // create commands
+    // 5 = left bumper
+    // 6 = right bumper
+
+    // Intake is toggled when left bumper is pressed
+    Trigger flipTrigger = m_driverHID.button(5);
+    flipTrigger.onTrue(m_intake.new FlipIntake(m_intake));
+
+    // Intake runs FORWARD when right trigger is pressed
+    Trigger runIntakeForwardsTrigger = m_driverHID.axisGreaterThan(3, Constants.DEADZONE);
+    runIntakeForwardsTrigger.whileTrue(new IntakeCommand(m_intake, true));
+
+    // Intake runs BACKWARD when right bumper is pressed
+    Trigger runIntakeBackwardsTrigger = m_driverHID.button(6);
+    runIntakeBackwardsTrigger.whileTrue(new IntakeCommand(m_intake, false));
+
+    Trigger invertDrivetrainTrigger = m_driverHID.button(1)
+    .onTrue(m_tankDriveSubsystem.new SwitchDrivetrainInvert(m_tankDriveSubsystem));
+
+    // new Intake
+    // Intake is reversed when right bumper is pressed
+    SwitchIntakeDirection m_switchIntake = m_intake.new SwitchIntakeDirection(m_intake);
+    Trigger switchIntakeTrigger = m_driverHID.button(6);
+    switchIntakeTrigger.onTrue(m_switchIntake);
   }
-  class setupControls extends CommandBase {
-    @Override
-    public void execute() {
-      configureButtonBindings();
-    }
 
-    @Override
-    public boolean isFinished() {
-        return true;
-    }
+  public static RobotContainer getInstance() {
+    return m_robotContainer;
   }
 
-    private void configureButtonBindings() {
-      m_tankDriveSubsystem.setDefaultCommand(m_driveCommand);
-    }
-  
-    public static RobotContainer getInstance() {
-      return m_robotContainer;
+  private void invertDrivetrain(boolean invert) {
+    if (invert) {
+      m_tankDriveSubsystem.setInverted(true, true, true);
+      System.out.println("Invert On");
+    } else {
+      m_tankDriveSubsystem.setInverted(false, false, false);
+      System.out.println("Invert Off");
     }
-  
+  }
 
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=FUNCTIONS
-  public CommandGenericHID getController() {
-      return m_driverHID;
+  private void invertDrivetrain() {
+    if (m_tankDriveSubsystem.getInverted()) {
+      m_tankDriveSubsystem.setInverted(false, false, false);
+      System.out.println("Invert Off");
+    } else {
+      m_tankDriveSubsystem.setInverted(true, true, true);
+      System.out.println("Invert On");
```

Comment:
I believe this is redundant with the invert drivetrain command existing within the TankDriveSubsystem.

@tobyMickelson  or @CalebNeal07 (I don't remember who worked the original....) can you confirm?

Reply:
I took it out in the recent commit, along with all variables, functions etc. related to inverting the drivetrain.